help/destinations/catalog/email-marketing/sendgrid.md

```diff
@@ -0,0 +1,201 @@
+
+![](../../assets/catalog/email-marketing/sendgrid/15.jpg)
+
+![](../../assets/catalog/email-marketing/sendgrid/16.jpg)
+
+* Similarly we map the remaining AEP fields and attributes to the Sendgrid attributes. For e.g. we are mapping the person object's first name atrribute below
+
+![](../../assets/catalog/email-marketing/sendgrid/17.jpg)
+
+![](../../assets/catalog/email-marketing/sendgrid/18.jpg)
+
+* A preview of some of the mapping's is as below
+
+![](../../assets/catalog/email-marketing/sendgrid/19.jpg)
+
+* The comprehensive list of the mappings that would be required to be setup is listed below.
+
+| Name | Field | Type | Description | Limits |
+|---|---|---|---|---|
+| contacts | - | Array of Objects | One or more contacts objects that you intend to upsert. The available fields for a contact, including the required email field are described below | maxItems: 30000 and minItems: 2 |
+| | address_line_1 | String | The first line of the address | maxLength: 100 |
+| | address_line_2 | String | An optional second line for the address | maxLength: 100 |
+| | alternate_emails | Array of String | Additional emails associated with the contact | maxItems: 5 and minItems: 0 |
+| | city | String | The contact's city | maxLength: 60 |
+| | country | String | The contact's country. Can be a full name or an abbreviation | maxLength: 50 |
+| | email | String | The contact's primary email. This is a mandatory field and the value passed should be valid | maxLength: 254 |
+| | first_name | String | The contact's name | maxLength: 50 |
+| | last_name | String | The contact's family name | maxLength: 50 |
+| | postal_code | String | The contact's ZIP code or other postal code | |
+| | state_province_region | String | The contact's state, province, or region | maxLength: 50|
+| | line | String | The contact's Line account *(if any)* | |
+| | whatsapp | String | The contact's Whatsapp account *(if any)* | |
+| | facebook | String | The contact's Facebook account *(if any)* | |
+| | unique_name | String | | |
+| | phone_number | String | The contact’s phone number | |
+| | custom_fields | Object | Custom field created by the user using the Create Custom Fields Definition API. `This mapping is not supported.` | |
+
+* After completing the mappings, select *Next* to obtain the below review screen indicating the audience and destination are connected.
+
+![](../../assets/catalog/email-marketing/sendgrid/23.jpg)
+
+* Select *Finish* to complete the setup.
+
+![](../../assets/catalog/email-marketing/sendgrid/24.png)
+
+## Validate data within Sendgrid {#validate}
+
+* To validate that you have correctly set up the destination, navigate to the list of destinations. Ensure that the destination you created is visible and the status is enabled.
+
+![](../../assets/catalog/email-marketing/sendgrid/25.jpg)
+
+![](../../assets/catalog/email-marketing/sendgrid/26.jpg)
+
+* Switch to the *Activation data* tab.
```

Comment:
use bold please for UI elements that users are interacting with. 
also, it is not clear to me what we are instructing the user to do here? once in the Activation data tab, should they select the segment name to get to the screen shown just below? let's be more explicit, please, and not leave users guessing.